Containers/nextcloud/Dockerfile

```diff
@@ -252,3 +253,5 @@ RUN echo "root:$(openssl rand -base64 12)" | chpasswd
 
 USER www-data
 ENTRYPOINT ["/start.sh"]
+
+HEALTHCHECK CMD (cgi-fcgi -bind -connect localhost:9000 && curl -skI localhost:7867) || exit 1
```

Comment:
maybe? Or is that not safe enough?

```suggestion
HEALTHCHECK CMD (nc -z localhost 9000 && curl -skI localhost:7867) || exit 1
```

Reply:
This also works
I don't know if the cgi-fcgi check is better, what's your opinion?

Reply:
I would use netcat for now. We already have too much dependencies inside the nextcloud container...

Reply:
done